src/component-library/Pages/Login/Login.tsx

```diff
@@ -171,7 +174,11 @@ export const Login = ({
                         setShowUserPassLoginForm(!showUserPassLoginForm)
                     }
                     } />
-
+                    <form
+                        onSubmit={(e) => {
```

Comment:
If we are not doing anything except "preventDefault" while submitting, we should remove the `onSubmit` attribute altogether